### ISSUE TITLE
Script entrypoints listed twice in build output

### ISSUE THREAD

**Issue by aklinker1** [aklinker1/wxt#29]
### Dev

```
WXT 0.1.2                                                                               9:55:24 AM
✔ Started dev server @ http://localhost:3001                                            9:55:25 AM
ℹ Pre-rendering chrome-mv3 for development with Vite 4.3.9                              9:55:25 AM
✔ Built extension in 406 ms                                                             9:55:25 AM
  ├─ .output/chrome-mv3/manifest.json               792 B                               9:55:25 AM
  ├─ .output/chrome-mv3/example.html                450 B   
  ├─ .output/chrome-mv3/options.html                597 B   
  ├─ .output/chrome-mv3/popup.html                  468 B   
  ├─ .output/chrome-mv3/sandbox.html                450 B   
  ├─ .output/chrome-mv3/background.js               14.31 kB
  ├─ .output/chrome-mv3/background.js               14.31 kB
  ├─ .output/chrome-mv3/chunks/example-07899964.js  1.71 kB 
  ├─ .output/chrome-mv3/chunks/options-07899964.js  1.71 kB 
  ├─ .output/chrome-mv3/content-scripts/overlay.js  10.34 kB
  ├─ .output/chrome-mv3/content-scripts/overlay.js  10.34 kB
  ├─ .output/chrome-mv3/assets/overlay.css          25 B    
  ├─ .output/chrome-mv3/icon/128.png                12.67 kB
  ├─ .output/chrome-mv3/icon/16.png                 698 B   
  ├─ .output/chrome-mv3/icon/32.png                 1.63 kB 
  ├─ .output/chrome-mv3/icon/48.png                 2.94 kB 
  └─ .output/chrome-mv3/icon/96.png                 8.08 kB 
Σ Total size: 81.53 kB
```

### Build

```
WXT 0.1.2                                                                               9:56:25 AM
ℹ Building chrome-mv3 for production with Vite 4.3.9                                    9:56:26 AM
✔ Built extension in 789 ms                                                             9:56:26 AM
  ├─ .output/chrome-mv3/manifest.json               520 B                               9:56:26 AM
  ├─ .output/chrome-mv3/example.html                297 B   
  ├─ .output/chrome-mv3/options.html                408 B   
  ├─ .output/chrome-mv3/popup.html                  301 B   
  ├─ .output/chrome-mv3/sandbox.html                297 B   
  ├─ .output/chrome-mv3/background.js               10.44 kB
  ├─ .output/chrome-mv3/background.js               10.44 kB
  ├─ .output/chrome-mv3/chunks/example-e26d7e02.js  10.44 kB
  ├─ .output/chrome-mv3/chunks/options-526ffc50.js  13.24 kB
  ├─ .output/chrome-mv3/content-scripts/overlay.js  10.34 kB
  ├─ .output/chrome-mv3/content-scripts/overlay.js  10.34 kB
  ├─ .output/chrome-mv3/assets/index-bad85938.css   34 B    
  ├─ .output/chrome-mv3/assets/overlay.css          25 B    
  ├─ .output/chrome-mv3/assets/popup-a1f3d880.css   25 B    
  ├─ .output/chrome-mv3/icon/128.png                12.67 kB
  ├─ .output/chrome-mv3/icon/16.png                 698 B   
  ├─ .output/chrome-mv3/icon/32.png                 1.63 kB 
  ├─ .output/chrome-mv3/icon/48.png                 2.94 kB 
  └─ .output/chrome-mv3/icon/96.png                 8.08 kB 
Σ Total size: 93.16 kB                                                                  9:56:26 AM
✔ Finished in 868 ms                                                                    9:56:26 AM
```

**Comment by aklinker1** [comment]
Git bisect shows it was introduced in #23

**Comment by aklinker1** [comment]
Fixed by 49965e7786a0e129d50028f8d28dd658c55e8301. Turns out the config hook for vite plugins merges the returned config with the user config, it doesn't replace the user config... So that simplifies things lol.
